kivy/uix/scrollview.py

```diff
@@ -661,6 +666,18 @@ def on_scroll_start(self, touch, check_children=True):
         if scroll_type == ['bars'] and not in_bar:
             return self.simulate_touch_down(touch)
 
+        if in_bar:
+            if (touch.ud['in_bar_y'] and not
+                    self._touch_in_handle(
+                        self._handle_y_pos, self._handle_y_size, touch)):
+                self.scroll_y = (touch.y - self.y) / self.height
+                return True
```

Comment:
I think it works better if you remove both of these `return True` statements. That way it jumps to the new position, and if you keep holding it will let you continue scrolling in that area.